client/src/components/AboutUs/AboutUs.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {Fade} from 'react-reveal'
+import { aboutusData, illustration } from '../../SpeakEd'
+// import DisplayLottie from '../../components/displayLottie/DisplayLottie'
+import Aboutus from '../../assets1/aboutus.svg';
+// import aboutus from '../../virtue'
```

Comment:
What is the use of `virtue` here? The JSON file is empty

Reply:
i will check it out

Reply:
@HarshCasper give me one day time , I will complete the issues!! , i had some exams! please give some time!!